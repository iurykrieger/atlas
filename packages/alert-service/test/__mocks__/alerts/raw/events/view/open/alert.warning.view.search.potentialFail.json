{
	"action": "Create",
	"alert": {
		"alertId": "7ddbe5c3-490a-41d6-b4e3-e15191aafbb8-1541881980135",
		"message": "[DC][prophetheus] search_views {\"apiKey\": \"cursoslivresead\"}",
		"tags": [
			"dc-prophetheus",
			"atd",
			"int"
		],
		"tinyId": "7409",
		"entity": "",
		"alias": "[DC][prophetheus] search_views {\"apiKey\": \"cursoslivresead\"}",
		"createdAt": 1535461323437,
		"details": {
			"checkTask": "search_views",
			"link": "http://prophetheus.dc.chaordicsystems.com:5000/details?task=search_views#group=%7B%22apiKey%22%3A+%22cursoslivresead%22%7D&start=1541874771.396501&end=-1",
			"serie": "{\"apiKey\": \"cursoslivresead\"}"
		},
		"updatedAt": 1535461323437000000,
		"username": "Alert API",
		"userId": "",
		"description": "[DC][prophetheus] search_views {\"apiKey\": \"cursoslivresead\"}",
		"teams": [
			"afa919f9-5c24-4b55-b60e-2548bdda7abf",
			"e4e9ae6d-cec6-4cd1-9fb1-a4c4a710fc83"
		],
		"priority": "P3",
		"source": "dc-ops"
	},
	"source": {
		"type": "API"
	},
	"integrationName": "Webhook-TAM",
	"integrationId": "75d1e86f-475f-484f-8b6f-0a3a276de9b5",
	"integrationType": "Webhook"
}
