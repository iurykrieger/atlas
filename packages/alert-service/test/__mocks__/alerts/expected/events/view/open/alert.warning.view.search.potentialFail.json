{
	"id": "[DC][prophetheus] search_views {\"apiKey\": \"cursoslivresead\"}",
	"status": "open",
	"apiKey": "cursoslivresead",
	"type": "alert.warning.view.search.potentialFail",
	"data": {
		"alertId": "7ddbe5c3-490a-41d6-b4e3-e15191aafbb8-1541881980135",
		"description": "Prophetheus: http://prophetheus.dc.chaordicsystems.com:5000/details?task=search_views#group=%7B%22apiKey%22%3A+%22cursoslivresead%22%7D&start=1541874771.396501&end=-1 \n [DC][prophetheus] search_views {\"apiKey\": \"cursoslivresead\"}",
		"tags": [
			"dc-prophetheus",
			"atd",
			"int"
		],
		"tinyId": "7409",
		"entity": "",
		"createdAt": 1535461323437,
		"details": {
			"checkTask": "search_views",
			"link": "http://prophetheus.dc.chaordicsystems.com:5000/details?task=search_views#group=%7B%22apiKey%22%3A+%22cursoslivresead%22%7D&start=1541874771.396501&end=-1",
			"serie": "{\"apiKey\": \"cursoslivresead\"}"
		},
		"updatedAt": 1535461323437000000,
		"username": "Alert API",
		"userId": "",
		"message": "[DC][prophetheus] search_views {\"apiKey\": \"cursoslivresead\"}",
		"teams": [
			"afa919f9-5c24-4b55-b60e-2548bdda7abf",
			"e4e9ae6d-cec6-4cd1-9fb1-a4c4a710fc83"
		],
		"priority": "P3",
		"source": "dc-ops"
	}
}
